src/org/labkey/targetedms/parser/PrecursorChromInfo.java

```diff
@@ -39,7 +39,7 @@
  */
 public class PrecursorChromInfo extends AbstractChromInfo implements Comparable<PrecursorChromInfo>
 {
-    private static final Logger LOG = LogManager.getLogger(PrecursorChromInfo.class);
+    private static final Logger LOG = LogHelper.getLogger(PrecursorChromInfo.class, "Messages about fetching chromatograms, either from the cache-in memory, or from S3/local files.");
```

Comment:
cache-in memory? What is that?  Is that "cache (in memory)" or "in-memory cache"?